src/libraries/Microsoft.PowerFx.Interpreter/Values/LambdaFormulaValue.cs

```diff
@@ -34,6 +34,19 @@ public async ValueTask<FormulaValue> EvalAsync()
             return await EvalInRowScopeAsync(_context).ConfigureAwait(false);
         }
 
+        public async ValueTask<FormulaValue> EvalInRowScopeAsync(RecordValue rowScope)
```

Comment:
>EvalInRowScopeAsync [](http://example.com/codeflow?start=45&length=19)

Can we do something like this?  To remove EvalVisitor/Context from the  FunctionInvokeInfo.